chainercv/links/utils/bbox/bbox_overlap.py

```diff
@@ -0,0 +1,42 @@
+from chainer import cuda
+
+
+def bbox_overlap(bbox, query_bbox):
```

Comment:
This function is very general. I think it is better to change the names of arguments more general. For example, 'bbox_a' and `bbox_b`. In this case, we can use `area_a` and `area_b` for the names of variables.